### ISSUE TITLE
## Make changes to your code.

### ISSUE THREAD

**Issue by theBwoyProgrammer** [alphayowakarindi/annual-diabetes-walk#2]
It should be nice to add the value of the alt attribute to improve the accessibility of the website.
https://github.com/alphayowakarindi/annual-diabetes-walk/blob/43ce7f4df7c68048e838b6b45fdf7a89bda96452/main.js#L62-L63

**Comment by Nedramrez** [comment]
https://github.com/alphayowakarindi/annual-diabetes-walk/blob/43ce7f4df7c68048e838b6b45fdf7a89bda96452/index.html#L129-L131
- Please add `alt `text to your images.

**Comment by TemesgenA** [comment]
Great work so far.
#### Features to improve:
The gap between the main program cards is wider, so I think you can reduce it so that you can display more cards in a row.
https://github.com/alphayowakarindi/annual-diabetes-walk/blob/43ce7f4df7c68048e838b6b45fdf7a89bda96452/style.css#L527-L532
